README.md

```diff
@@ -255,12 +255,17 @@ http://www.thoughtworks.com/insights/blog/test-assertions-how-do-they-work
 
 ## Requirements
 
-> Given a **Price** and amount of **Cash** from Customer
-> Return: **Change** to customer (*in notes and coins*).
+As a customer, I want to buy a selected item from the **vending machine** and see what my change 
+is as a **result** into the various **coins** so that I can select one of the options and receive my change.
 
-Essentially we are building a *simple* **calculator** that *only does* **subtraction**
-(Price - Cash = Change), but also splits the **result** into the various **coins**.
+Acceptance criteria:
+- A success call of a function `getChange` should return the change value in the various **coins** available
```

Comment:
@emersonmellado your English is _superb_! (_keep up the great work!_)
Did you know **40%** of people in the world **_only_ know One Language**! 
See: http://ilanguages.org/bilingual.php or PDF: [bilingual.pdf](https://github.com/dwyl/learn-tdd/files/1199698/multi-lingual-people.pdf)
And given that you know a couple of _programming_ languages you can be considered "***multilingual***" which automatically makes you one of the most _capable_ humans on the planet! 😉